docs/materials/osg/ex1-submit-refresher.md

```diff
@@ -34,8 +34,8 @@ Now, let’s try to remember some basic HTCondor ideas from earlier today:
 
         Requirements = (HAS_CVMFS_oasis_opensciencegrid_org =?= TRUE)
 
-    Try to do this step without looking at materials from earlier today.
-    But if you are stuck, see [today’s exercise 2.2](/materials/htc/part2-ex2-queue-n.md).
+    Try to do this step without looking at materials from the earlier exercises.
+    But if you are stuck, see [Exercise 2.2](/materials/htc/part2-ex2-queue-n.md).
```

Comment:
“Exercise 2.2” is technically (and actually) not a unique identifier — probably needs to be “HTC Exercise 2.2”.